src/main/java/com/dragonballs/services/user/UserValidator.java

```diff
@@ -0,0 +1,37 @@
+package com.dragonballs.services.user;
+
+import com.dragonballs.entities.User;
+import com.dragonballs.exceptions.UserException;
+
+import java.util.regex.Pattern;
+
+public class UserValidator {
+
+    private static final String USERNAME_BOTTOM_MARGIN = "Username must be longer than 5 symbols";
+    private static final int USERNAME_BOTTOM_MARGIN_VALUE = 5;
+
+    private static final String PASSWORD_MARGIN = "Password must be between 7 and 12";
```

Comment:
Use USERNAME_BOTTOM_MARGIN_VALUE  and PASSWORD_BOTTOM_MARGIN_VALUE  to construct it - no need to specify the same numbers multiple times